### PR TITLE
Chapter2/homework/4

### PR DESCRIPTION
**Pull request by littleNoodle**

### CODE REVIEW

File: Src/BootCamp.Chapter/ILog.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    interface ILog
+    {
+        public void WriteMessageToFile(string messge);
+        public void WriteMessageToConsole(string messge);
```
Comment (Almantask) [inline Src/BootCamp.Chapter/ILog.cs]: Purpose of an interface should provide a contract of what can be done.
You can log- that's what yuo can do.
ToFile or ToConsole- is a detail- not something that you should concern an interface with.

File: Src/BootCamp.Chapter/Logger.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using static System.Console;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    class Logger: ILog
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Logger.cs]: You need two types of loggers: FileLogger and ConsoleLogger

File: Src/BootCamp.Chapter/Calculate.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                string message = ($"{name} is {age} years old, weight is {weight} kg, height is {height} cm, BMI is {bmi}");
+                log.WriteMessageToFile(message);
+                log.WriteMessageToConsole(message);
+
+                Console.WriteLine("Once again? Y/N");
+                var input = Console.ReadKey();
+                if(input.KeyChar.ToString().ToUpper() == "Y")
+                {
+                    Console.WriteLine();
+                    CalculateBMI(log);
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Calculate.cs]: Split into steps

**Review by Almantask — CHANGES_REQUESTED** [review]

File: Src/BootCamp.Chapter/ILog.cs (hunk 1 above)
Comment (littleNoodle) [reply Src/BootCamp.Chapter/ILog.cs]: Thanks for sharing! I have more sense of interface^_^

**Pull request closed without merging**